src/test/java/edu/ksu/canvas/attendance/repository/ReportRepositoryITest.java

```diff
@@ -44,11 +41,15 @@
     
     @Autowired
     private AttendanceCourseRepository courseRepository;
+
+    @Autowired
+    private AttendanceSectionRepository sectionRepository;
     
     @Autowired
     private ReportRepository reportRepository;
     
     private AttendanceCourse existingCourse;
+    private AttendanceSection existingSection;
```

Comment:
looks like this field was added and isn't used. I agree with sonar bot's suggestion.